.github/workflows/lint-php.yml

```diff
@@ -0,0 +1,58 @@
+name: Lint PHP
+on:
+  pull_request:
+    types: [ opened, edited, reopened, ready_for_review ]
```

Comment:
We may want to consider adding the `synchronize` activity type? 
See: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Ultimately, I think we want to move this workflow to our reusable workflows: https://github.com/newfold-labs/workflows
This will allow us to make a single change and update all repos that use it.